src/Generated/.clang-tidy

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Is it auto-generated file? Why is it under Generated/ path?

Reply:
No it's not auto generated, but it's a trick to ignore generated file under a directory